src/community/geogig/src/main/java/org/geogig/geoserver/gwc/TruncateTilesOnUpdateRefHook.java

```diff
@@ -77,7 +77,7 @@ public boolean appliesTo(Class<? extends AbstractGeoGigOp<?>> clazz) {
          * Store the ref name and its old value in the command's user properties to be used in the
          * post-hook if the operation was successful
          */
-        Map<String, Object> params = Scripting.getParamMap(command);
+        Map<String, Object> params = null;
         String refName = (String) params.get("name");
```

Comment:
This is now an obvious null pointer exception; fix it, and add a test case, since it evidently isn't getting tested.